cpp/graph_algorithms_module/test/apollo_runs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
```

Comment:
This entire file should be deleted.

Reply:
sorry for inconveniences, will be done asap